lib/d/Makefile.am

```diff
@@ -136,54 +110,22 @@ docs: $(d_publicsources) src/thrift/index.d
 #
 # Hook custom library targets into the automake all/install targets.
 #
-all-local: $(all_targets)
+all-local:
+	dub build --config=library --arch=x86_64 --build=debug --compiler=dmd
```

Comment:
Multiple problems:

1. All builds are 64-bit.
2. All builds are debug.
3. All builds require dmd.  autoconf detects any of the three dlang compilers that might be installed.